config.yml.erb

```diff
@@ -628,11 +628,8 @@ jwks_data:
 jwk_private_key_data: !Secret
 
 # Credentials for OpenAI API
-openai_chat_completion_api_key: !Secret
-openai_chat_completion_org_id: !Secret
 openai_evaluate_rubric_api_key: !Secret
-openai_evaluate_rubric_org_id:
```

Comment:
This isn't set in Secrets Manager and I think was just copy/paste per the PR below, so I deleted it to tidy up. But double checking with @davidsbailey just in case :)

https://github.com/code-dot-org/code-dot-org/pull/53997

Reply:
looks like this is no longer used, so ok to remove!